sites/fast-website/src/app/data/feature.data.ts

```diff
@@ -1,65 +1,133 @@
+interface FeatureLink {
+    anchorText: string;
+    url: string;
+}
+
 export interface FeatureCardData {
-    body: string;
-    documentationLink: string;
-    githubLink: string;
     header: string;
+    body: string;
+    links?: FeatureLink[];
 }
 
 export const featureCardData: FeatureCardData[] = [
     {
+        header: "Composition",
         body:
             "Create new component compositions by nesting, styling base components, and passing unhandled props. The combinations are endless.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Composition",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
```

Comment:
do we not have URLs for this? If they're yet to be defined, can we write a TODO comment at the top of this const as a reminder to fill these out?

Reply:
The features in the design are a direct copy of the existing features listed on the current site for the React based components.  Note the description text in feature 3 specifically referencing React.  We've got github links that tie to a few of these, but no matching documentation page comparable to the one associated with the React components yet.  I'll add a TODO.  Thanks Jane.

Reply:
I'm going to hold until we get correct text. At this point, I don't see value in adding content debt of this size.

Reply:
@freefaller69 - If you can drop all the secondary anchors from this I'm fine to have this go in. I can take a follow-up to fix the actual text.

Reply:
Secondary anchors removed.

<img width="1209" alt="Screen Shot 2020-06-23 at 10 34 53 AM" src="https://user-images.githubusercontent.com/26206206/85437006-c4a57900-b53e-11ea-81df-0d1fa88f2d61.png">

Reply:
Sorry, to be clear - just the second anchor. I'd expect each one to have one, and I'll do a run through to edit the text and desitination (anchors missing on composition, design systems, technology).

Reply:
My bad.  Replaced primary anchors.  They have TODOs on them as well per Jane's comment above.

<img width="1209" alt="Screen Shot 2020-06-23 at 11 35 59 AM" src="https://user-images.githubusercontent.com/26206206/85443477-2c12f700-b546-11ea-93b4-dbc5808c0761.png">